aten/src/ATen/templates/Functions.h

```diff
@@ -146,4 +146,7 @@ inline bool is_complex(const Tensor& tensor) {
   return tensor.is_complex();
 }
 
+inline bool is_floating_type(const Tensor& tensor) {
```

Comment:
should be `inline bool is_floating_point(const Tensor& tensor) {`

Also I'd suggest locally compiling the changes before pushing them :)